.github/workflows/CI.yml

```diff
@@ -116,7 +116,7 @@ jobs:
         path: |
           ~/go/pkg/mod
           ~/.cache/go-build
-        key: go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.sum') }}
+        key: v1-go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.sum') }}
```

Comment:
What was this change for?

Reply:
Mostly naming convention. IMO it is easier for dev to test the CI without thinking another name.

Reply:
I don't understand the convention. What is the v1 for?

Reply:
Its stand for version 1. One of the reason I used this is to easier delete or using another build cache, by only changing the v1 to v2 instead of thinking another name or delete some words.  Also [there is currently no delete cache ](https://stackoverflow.com/questions/63521430/clear-cache-in-github-actions) .